workflow/static/js/timesheet_autosave.js

```diff
@@ -1,4 +1,13 @@
-// Debounce function to reduce frequent server calls
+let deletedEntries = [];
+
+function markEntryAsDeleted(entryId) {
+    if (entryId) {
```

Comment:
Ok.

I thought we were simply getting the data from the form and essentially replacing the data in the data model from the form?